### ISSUE TITLE
Error while setting up a new Analogjs project with Nx 17

### ISSUE THREAD

**Issue by luishcastroc** [analogjs/analog#713]
### Please provide the environment you discovered this bug in.

N/A

### Which area/package is the issue in?

create-analog

### Description

When a new project needs to be created using Nx the command to use is:

```bash
npx create-nx-workspace@latest --preset=@analogjs/platform
```

after following the instructions an error is presented.

### Please provide the exception or error you saw

```true
Angular has not been installed yet. Creating an Angular application

 >  NX   As of Nx 16.0.0 the @nrwl scope has been replaced with the @nx scope. Please use @nx scope to install version 17.0.1

   Pass --verbose to see the stacktrace.


 >  NX   Failed to apply preset: @analogjs/platform

   See above
```

### Other information

node 18.17.1 is used.

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [ ] No

**Comment by Pascalmh** [comment]
Should be fixed by #715

**Comment by brandonroberts** [comment]
Fixed in https://github.com/analogjs/analog/pull/715 and released in `0.2.14`
